radicle-cli/src/commands/delegate.rs

```diff
@@ -0,0 +1,122 @@
+use std::ffi::OsString;
+use std::str::FromStr;
+
+use anyhow::{anyhow, Context as _};
+
+use radicle::identity::Id;
+use radicle_crypto::PublicKey;
+
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
+mod add;
+mod list;
+mod remove;
```

Comment:
Probably these should be under a `delegate/` folder

Reply:
Whoops, think this was an IDE bug when adding the files from my Emacs. Guess it doesn't play well with that `path` config